chains.config.cjs

```diff
@@ -0,0 +1,70 @@
+// chain configs in ocean.js ConfigHelperConfig format
+// see: https://github.com/oceanprotocol/ocean.js/blob/e07a7cb6ecea12b39ed96f994b4abe37806799a1/src/utils/ConfigHelper.ts#L8
```

Comment:
also for this chains if not suppoerted by ocean js we need a pr but should be fine afk, and use the sdk for fetching this data